Makefile

```
@@ -393,6 +401,17 @@ docker-multiarch-build: go-generate generate-config-file
 		--push \
 		-t $(OPERATOR_IMAGE)
 
+docker-multiarch-build-dockerhub:
+	@ hack/docker.sh -l -m $(OPERATOR_DOCKERHUB_IMAGE)
+	docker buildx build . \
+		--progress=plain \
+		--build-arg GO_LDFLAGS='$(GO_LDFLAGS)' \
+		--build-arg GO_TAGS='$(GO_TAGS)' \
+		--build-arg VERSION='$(VERSION)' \
+		--platform linux/amd64,linux/arm64 \
+		--push \
+		-t $(OPERATOR_DOCKERHUB_IMAGE)
```

Why rebuild an image? Couldn't we just tag and push?
```suggestion
	docker tag $(OPERATOR_IMAGE) $(OPERATOR_DOCKERHUB_IMAGE)
	docker push $(OPERATOR_DOCKERHUB_IMAGE)
```

---

I initially attempted this, and ran into issues, but my approach was incorrect, and just telling buildx to tag 2x tags, and the `--push` flag makes it work without issues.  I've pushed this change.